cmd/user-agent/src/pages/chapi/wallet/didcomm/blindedRouter.js

```diff
@@ -45,11 +44,17 @@ export class BlindedRouter {
         })
 
         // send message to router requesting peerDID
-        // TODO response peer DID from previous step to be sent to router
-        this.mediator.requestDID()
+        // TODO response peer DID from previous step to be sent to router #409
+
+        // TODO temporarily send myDID to test response (to test e2e) #409
```

Comment:
_getPeerDID is sending `TheirDID`

Reply:
fixed, just a temporary code to test integration with router